tests/extendrtests/README.md

```diff
@@ -18,10 +18,10 @@ The Cargo.toml file hardcodes the relative path of the `extendr` libraries, thus
 in order to check this project locally you need to run:
 
 ```r
-devtools::check(check_dir = "check")
+rcmdcheck::check(check_dir = "check")
 ```
 
 This is necessary so the relative path in Cargo.toml points to the correct
 location. **Note**: Clicking 'Check' in RStudio will not work because it runs
-`devtools::check()` with no arguments and the check are then run in a temporary
+`devtools::check()` with no arguments and the checks are then run in a temporary
```

Comment:
Also, if you change code fragment above to `rcmdcheck::rcmdcheck`, do it also here.

Reply:
RStudio actually calls `devtools::check()` so I'm leaving that but added parenthesis explaining. `devtools::check` is really a wrapper for `rcmdcheck::rcmdcheck` so there shouldn't be any differences in the end.

Reply:
@dfalbel,
My mistake, you are correct.